invenio_cli/commands/local.py

```diff
@@ -84,16 +84,17 @@ def update_statics_and_assets(self, force, debug=False, log_file=None):
         Needed here (parent) because is used by Assets and Install commands.
         """
         # Commands
-        pkg_man = self.cli_config.python_package_manager
-        ops = [pkg_man.run_command("invenio", "collect", "--verbose")]
+        py_pkg_man = self.cli_config.python_package_manager
+        js_pkg_man = self.cli_config.javascript_package_manager
+        ops = [py_pkg_man.run_command("invenio", "collect", "--verbose")]
 
         if force:
-            ops.append(pkg_man.run_command("invenio", "webpack", "clean", "create"))
-            ops.append(pkg_man.run_command("invenio", "webpack", "install"))
+            ops.append(py_pkg_man.run_command("invenio", "webpack", "clean", "create"))
+            ops.append(py_pkg_man.run_command("invenio", "webpack", "install"))
         else:
-            ops.append(pkg_man.run_command("invenio", "webpack", "create"))
+            ops.append(py_pkg_man.run_command("invenio", "webpack", "create"))
         ops.append(self._statics)
-        ops.append(pkg_man.run_command("invenio", "webpack", "build"))
+        ops.append(py_pkg_man.run_command("invenio", "webpack", "build"))
```

Comment:
there is still a good amount of optimization potential by calling the `invenio` command only once and thus saving all the redundant application initializations which eat up a lot of time.
one way to move forward is to implement a single larger command (e.g. in `invenio-app-rdm`) that performs all the steps currently implemented here, based on the provided CLI args.
@ntarocco has pointed out to me that something very much like that has already been done in [Invenio-App-ILS](https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/cli.py#L1567)

Reply:
an option would be to use a `rpc-server` implementation. a WIP would be [here](https://github.com/inveniosoftware/invenio-cli/pull/387) for `invenio-cli` combined with [PR](https://github.com/inveniosoftware/invenio-app/pull/99) from `invenio-app`